src/bosh-monitor/lib/bosh/monitor/metric.rb

```diff
@@ -12,5 +12,14 @@ def initialize(name, value, timestamp, tags)
       @timestamp = timestamp
       @tags = tags
     end
+
+    def to_hash
+      {
+          :name => @name,
+          :value => @value.to_s,
```

Comment:
seems a bit weird to cast it to string given that some of these values are ints: https://github.com/cloudfoundry/bosh/blob/c6c419f9eff844c35c8c7307e977427f05a23a51/src/bosh-monitor/lib/bosh/monitor/events/heartbeat.rb#L107